en/micro-integrator/docs/setup/observability/setting-up-classic-observability-deployment.md

```diff
@@ -104,7 +117,65 @@ Follow the steps below to enable statistics for the **endpoint** artifacts:
+     
+### Enabling load balancing Data Agent for Publishing to Analytics
+
+You can send events to multiple Analytics servers either by sending the same event to many Analytics servers or by load balancing events among a set of servers. This handles the fail-over problem. When events are load balanced within a set of servers and if one receiver cannot be reached, events are automatically sent to the other available and active Analytics servers. The following scenarios are covered in this section.
+
+#### Load balancing across group of servers 
+
+For this functionality, include the server URL in the Data Agent as a general Analytics receiver URL. The URL should be entered in a comma separated format as shown below.
+
+eg: `tcp://<Analytics-1>:<port>,tcp://<Analytics-2>:<port>,tcp://<Analytics-3>:<port>`
+
+sample: `tcp://10.100.2.32:7611, tcp://10.100.2.33:7611, tcp://10.100.2.34:7611`
+
+![lb events to servers](../../assets/img/ei-analytics/ob-lb-events-to-servers.png)
+    
+This handles failover as follows: If Analytics Receiver-1 is marked as down, then the Data Agent will send the data only to the Analytics Receiver-2 and Analytics Receiver-3 in a round robin manner. When the Analytics Receiver-1 becomes active after some time, the Data Agent automatically detects it, adds it to the operation, and again starts to load balance between all three receivers. This functionality significantly reduces the loss of data and provides more concurrency.
+
+#### Load balancing across multiple groups of servers  
+
+In this setup, there are two sets of servers that are referred to as set-A and set-B. 
+You can send events to both the sets. You can also carry out load balancing for both sets as mentioned in load balancing between a set of servers. 
+This scenario is a combination of load balancing between a set of servers and sending an event to several receivers. 
+An event is sent to both set-A and set-B. Within set-A, it will be sent either to Analytics A1 or Analytics A2. 
+Similarly within set-B, it will be sent either to Analytics B1 or Analytics B2. 
+In the setup, you can have any number of sets and any number of servers as required.
+    ![lb events to set of servers](../../assets/img/ei-analytics/ob-lb-to-sets-of-servers.png)
+    
+Similar to the other scenarios, you need to describe the server URLs as the receiver URL in the Micro Integrator configuration. The sets should be specified within curly braces separated by commas. Furthermore, each receiver that belongs to the set should be within the curly braces and with the receiver URLs in a comma separated format. The receiver URL format is given below.
+
+eg: `{tcp://Analytics-A1:port, tcp://Analytics-A2:port},{tcp://Analytics-B1:port, tcp://Analytics-B2:port}`
+
+sample: `{tcp://10.100.2.32:7611, tcp://10.100.2.33:7611}, {tcp://10.100.2.34:7611, tcp://10.100.2.35:7611}`
+
+#### Sending all the events to several analytics servers
```

Comment:
#### Sending all events to several analytics servers

This setup involves sending all the events to more than one Analytics server. 
This approach is useful when you want to have multiple Analytics servers to analyze the same events simultaneously. 
For example, as shown below, you can configure the Micro Integrator to publish the same event to both Analytics servers at the same time. 

 ![all events to all servers](../../assets/img/ei-analytics/ob-all-events-to-all-servers.png)

The Analytics receiver URL should be configured with the following format in the Micro Integrator: `{tcp://Analytics-1>:<port>}, {tcp://Analytics-2>:<port>}, {tcp://<Analytics-3>:<port>}`

Example configuration in the `deployment.toml` file of the Micro Integrator:

```toml
[monitoring]
ei_analytics.server_url = "{tcp://10.100.2.32:7611},{ tcp://10.100.2.33:7611}, {tcp://10.100.2.34:7611}"
ei_analytics.auth_server_url = "{tcp://10.100.2.32:7612},{ tcp://10.100.2.33:7612}, {tcp://10.100.2.34:7612}"
ei_analytics.username = "admin"
ei_analytics.password = "admin"  
```